src/functions/IFunction.java

```diff
@@ -0,0 +1,6 @@
+package functions;
+
+public interface IFunction {
```

Comment:
May need to change to a subscribe pattern where all subscribed organs receive functionality updates from the function subscribed to.

```
[
  b:Leaf( // check is subscribed ) ::> // some update
]
```

Reply:
Could also jus require organ to be passed in

Reply:
Could split into two different kinds of functions:
1. Subscribe "world" functions (that follow global execution)
2. Organ functions (only execute on the organ it is part of)